app/lib/account/backend.dart

```diff
@@ -86,7 +86,12 @@ UserSessionData? get userSessionData =>
 /// When no associated User entry exists in Datastore, this method will create
 /// a new one. When the authenticated email of the user changes, the email
 /// field will be updated to the latest one.
-Future<AuthenticatedUser> requireAuthenticatedUser(
+Future<AuthenticatedUser> requireAuthenticatedUser() async {
```

Comment:
```suggestion
Future<AuthenticatedUser> requireAuthenticatedWebUser() async {
```

Reply:
This is only for web users, right? (this can also be renamed in a follow-up, that only renames this).

Reply:
Correct. I've renamed it.